source/crud/tests/v2/unacknowledged-findOneAndReplace-hint-clientError.yml

```diff
@@ -0,0 +1,46 @@
+runOn:
+  # These tests assert that the driver raises a client-side error when the hint
+  # option is passed with an unacknowledged update operation against a server
+  # that does not support hints in update operations.
+  - { maxServerVersion: "4.3.0" }
```

Comment:
Optional, but if we want to switch on the server version that changed the wire version, maybe we should have this as maxServerVersion 4.3.3 (assuming 4.3.4 was the server version when the wire version was bumped from 8 to 9).

Though it might not matter. If servers on all platforms are sufficiently updated, drivers won't be testing against these intermediary releases. I'll leave it up to you, but it might be worth changing just for consistency.

Reply:
This is a great point! I'll change it.